src/devices.c

```diff
@@ -617,6 +626,7 @@ int Devices_Initialise(int *argc, char *argv[])
 				Log_print("\t-H3 <path>       Set path for H3: device");
 				Log_print("\t-H4 <path>       Set path for H4: device");
 				Log_print("\t-Hpath <path>    Set path for Atari executables on the H: device");
+				Log_print("\t-Hdevicename <X> Rename H: device for something different");
```

Comment:
IMHO, a better description could be like "Use this letter to access the Host device, instead of H:".

Reply:
How to do this best? As second commit or create new pull request with one commit? I am a kind of new in PullRequesting - what is the procedure with PR which have to be changed/fixed?